astro/src/content/blog/fusionauth-web-summit.mdx

```diff
@@ -397,16 +380,6 @@ Have fun, but it’s not a party. You can do that when you get home after landin
 
 You’ve read my whole guide, wow!!! So that gets you a ticket to our FusionAuth exclusive event on night one. Simply [register here](https://fusionauth.link/ws-blog) and I’ll personally send you an invite.
 
-**See Us at Web Summit**
-
-Please come and say hi to FusionAuth at our booth, or see me (Brendan) whilst I host the New Venture stage on Day 1 and the AI Summit on Day 2.
-
 **Don’t know FusionAuth?**
 
-FusionAuth adds login, registration, SSO, MFA, and a bazillion other features to your app in days - not months.
-
-[Click here](https://fusionauth.link/startups) to find out more and sign up to the waitlist for our exclusive program for early stage startups.
-
-
-
-
+FusionAuth adds login, registration, SSO, MFA, and a bazillion other features to your app in days - not months. [Get started](/get-started) for free.
```

Comment:
I still think we should have a big orange button for these kinds of CTAs. 

And that you should track internally which 'get started' links lead to contact us submissions. (Maybe you do and I don't know about it.)

But that's more of a grumpy old man comment than a showstopper.